hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/TestFSQueueConverter.java

```diff
@@ -149,20 +152,20 @@ public void testConvertQueueHierarchy() {
     converter.convertQueueHierarchy(rootQueue);
 
     // root children
-    assertEquals("root children", "admins,users,misc,default",
-        csConfig.get(PREFIX + "root.queues"));
+    assertEquals("root children", new String[]{"admins", "users", "misc", "default"},
```

Comment:
I will update this with assertArrayEquals as I was unaware of assertEquals is deprecated for array.